### PR TITLE
python3Packages.libtmux: fix disabledTestPaths in darwin

### LINKED ISSUE #221505: Build failure: python3Packages.libtmux
### Steps To Reproduce

The derivation failed to build on macOS at test stage

```
# nix-shell -p python310Packages.libtmux -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixos-unstable.tar.gz
```

### Build log

```
...
pythonRemoveBinBytecodePhase
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Check whether the following modules can be imported: libtmux
pytestCheckPhase
Executing pytestCheckPhase
Disabled tests path "test_test.py" does not exist. Aborting
error: builder for '/nix/store/1wbl1f7niy3dqz173dyy582nmx64mxzl-python3.10-libtmux-0.21.0.drv' failed with exit code 1;
       last 10 log lines:
       > running install tests
       > no Makefile or custom installCheckPhase, doing nothing
       > pythonCatchConflictsPhase
       > pythonRemoveBinBytecodePhase
       > pythonImportsCheckPhase
       > Executing pythonImportsCheckPhase
       > Check whether the following modules can be imported: libtmux
       > pytestCheckPhase
       > Executing pytestCheckPhase
       > Disabled tests path "test_test.py" does not exist. Aborting
       For full logs, run 'nix log /nix/store/1wbl1f7niy3dqz173dyy582nmx64mxzl-python3.10-libtmux-0.21.0.drv'.
```

### Additional context
Encounter this problem when build `tmuxp`, as `libtmux` is the dependency.

### Notify maintainers

@otavio

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 22.2.0, macOS 13.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.11.1`
 - channels(ethinx): `"darwin"`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

### PR DESCRIPTION
**Pull request by ethinx**
###### Description of changes

Closes #221505.

Change `disabledTestPaths` in the `libtmux` derivation.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by wegank — APPROVED** [review]

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -16946,5 +16946,11 @@
     githubId = 32876;
     name = "Diego Zamboni";
   };
+  ethinx = {
+    email = "eth2net@gmail.com";
+    github = "ethinx";
+    githubId = 965612;
+    name = "York Wong";
+  };
```
Comment (wegank) [inline maintainers/maintainer-list.nix:16954]: Please keep the list sorted, thanks!

Comment (ethinx) [reply maintainers/maintainer-list.nix]: now it should be at the right position, thanks @wegank

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 221506` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python311Packages.libtmux</li>
  </ul>
</details>
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>arsenal</li>
    <li>python310Packages.libtmux</li>
    <li>tmuxp</li>
  </ul>
</details>

**Comment by wegank** [comment]
Looks like `test_capture_pane_start` is flaky.

**Pull request merged**